examples/official-storybook/stories/core/parameters.stories.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+// We would need to add this in config.js idomatically however that would make this file a bit confusing
+import { addParameters } from '@storybook/react';
+
+addParameters({ globalParameter: 'globalParameter' });
+
+export default {
+  title: 'Core|Parameters',
+  decorators: [fn => fn({ parameters: { decoratorParameter: 'decoratorParameter' } })],
+  parameters: {
+    chapterParameter: 'chapterParameter',
+  },
+};
+
+// I'm not sure what we should recommend regarding propTypes? are they a good idea for examples?
+// Given we sort of control the props, should we export a prop type?
```

Comment:
Technically, story itself is not a React component: decorators call it as a function, and propTypes have no effect in that case

Reply:
In react they are eventually invoked as a React Component in the renderer though?

Reply:
Not the inner story function but `decoratedStory`